client_internals.go

```diff
@@ -85,6 +85,10 @@ func NewClientInternal(c *Client) *ClientInternal {
 	}
 }
 
+func (ci *ClientInternal) ProxyManagerProxies() map[string]interface{} {
```

Comment:
This looks like it is used only in a test. Could you move it to `client_internals_test_utils.go`?

Reply:
Yes, I moved it. [4f3ef73](https://github.com/hazelcast/hazelcast-go-client/pull/772/commits/4f3ef7364ca9128a95eefad10ab2899294a60607).